script.js

```diff
@@ -143,13 +154,76 @@ const confirmOfClearForm = () => {
     });
 };
 
+const valuesToForm = id => {
+  const values = $$("filmsDataTable").getItem(id);
+  $$("addFilmForm").setValues(values);
+};
+
+webix.DataStore.prototype.sorting.as.sortByFloat = (firstNum, secondNum) => {
```

Comment:
instead of parsing numbers on each sort (as well as when form rules are applied), you can parse formatted numbers in the `scheme` $init handler:

```
view:"datatable",
scheme:{
   $init:function(obj){
       obj.votes = obj.votes.replace(",", "");
   }
}
```

Reply:
Ready to request